### PR TITLE
Bug 1169637 - Fix failure in test_marketplace_packaged_app.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=676ff1271b2d49d9b4cbd053dc472ec5542a848f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=96b265b9efc50111c577a130281a2b8c1318ce14

File: tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py [hunk 1]
```diff
@@ -53,7 +53,6 @@ class SearchResults(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        self.wait_for_element_not_present(*self._search_results_loading_locator)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py]: nit: No need to define the constructor if we remove this line

File: tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py [hunk 2]
```diff
@@ -28,9 +28,9 @@ def search(self, term):
         import time
         time.sleep(10)
 
-        search_toggle = Wait(self.marionette).until(
+        search_toggle = Wait(self.marionette, timeout=30, interval=5).until(
             expected.element_present(*self._search_toggle_locator))
-        Wait(self.marionette).until(expected.element_displayed(search_toggle))
+        Wait(self.marionette, timeout=30, interval=5).until(expected.element_displayed(search_toggle))
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py]: What does it change to specify the interval here?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py]: Nothing really. Probably, this change isn't necessary, but when I tried to fix this issue, I had networking issues. I'll revert this change.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b4980d895c4bebdda45eb4125f44ecd04c9ce46

**Pull request merged**